fs/react-next-base
# Toast Notifier

# Summary

#### A brief description of the pull request.

#### Resolves: #59 

#### Screenshots in case of UI changes
![Peek 2020-08-04 10-57](https://user-images.githubusercontent.com/47859969/89269142-309cf600-d642-11ea-9e0d-8d302f2cbb3d.gif)

![Peek 2020-08-19 01-47](https://user-images.githubusercontent.com/47859969/90573005-06a90f00-e1be-11ea-8f3b-72b1f0a1842a.gif)


# Test plan

#### List of steps to manually test introduced functionality:

- Go to Login page
- try to sign in with non-existent login/password
- you should see error notifier

- Go to profile page
- update your name 
- you should see success message
